src/test/Fake.Core.IntegrationTests/SimpleHelloWorldTests.fs

```diff
@@ -47,13 +47,19 @@ let tests =
               prepare scenario
               let scenarioPath = resolvePath scenario ""
               // dotnet tool install --version 5.19.0-alpha.local.1 fake-cli --add-source /e/Projects/FAKE/release/dotnetcore/
+
+              // Work around https://github.com/dotnet/sdk/issues/40655 by specifying the tool manifest explicitly
```

Comment:
Some of tests in previous runs seemed to be failing with https://github.com/dotnet/sdk/issues/40655, whereby installing the dotnet tool in a subdirectory that contains a tools config file will ignore that file and use the one in the repo root, but then trying to use that tool *will* use the local config file and then fail.
That issue is marked as resolved, but it still seemed to be occuring in the SDK version used by the tests